docs/development/background/getting-started-5-quick-quiz.md

```diff
@@ -0,0 +1,42 @@
+# Electron Quick Quiz
+
+## What does a v8:Isolate correspond to?
+
+Isolate corresponds to a physical thread. Isolate : physical thread in Blink = 1 : 1. The main thread has its own Isolate. A worker thread has its own Isolate.
+
+## What does a v8:Context correspond to?
+
+Context corresponds to a global object (In case of a Frame, it's a window object of the Frame). Since each frame has its own window object, there are multiple Contexts in a renderer process. When you call V8 APIs, you have to make sure that you're in a correct context. Otherwise, v8::Isolate::GetCurrentContext() will return a wrong context and in the worst case it will end up leaking objects and causing security issues.
+
+## What is the difference between a static and shared library in C++?
+
+Static libraries, while reusable in multiple programs, are locked into a program at compile time. Dynamic, or shared libraries on the other hand, exist as separate files outside of the executable file.
+
+## What is the difference between a strong and weak reference? Explain. Give an example.
+
+Weak reference. In computer programming, a weak reference is a reference that does not protect the referenced object from collection by a garbage collector, unlike a strong reference.
+
+http://elliot.land/post/strong-vs-weak-references
+
+## What is libchromiumcontent?
```

Comment:
This changed in Electron `v4` and beyond; we no longer maintain libchromiumcontent as a separate library & use GN to build instead of gyp. This section will need to make this distinction.